pkg/agent/config.go

```diff
@@ -34,8 +34,9 @@ type Config struct {
 	// in the AgentID field of each flow. Accepted values are: any (default), ipv4, ipv6.
 	// If the AgentIP configuration property is set, this property has no effect.
 	AgentIPType string `env:"AGENT_IP_TYPE" envDefault:"any"`
-	// Export selects the flows' exporter protocol. Accepted values are: grpc (default), kafka,
-	// ipfix+udp, ipfix+tcp or direct-flp.
+	// Export selects the exporter protocol.
+	// Accepted values for Flows are: grpc (default), kafka, ipfix+udp, ipfix+tcp or direct-flp.
+	// Accepted values for Packets are: grpc (default) or tcp
```

Comment:
Previously, PCA was only supporting tcp and required its own configs prefixed with PCA_
Now [I'm trying to merge with flow configs](https://github.com/netobserv/netobserv-ebpf-agent/pull/291#discussion_r1521064391) and support both gRPC + tcp

Don't you think it's worth ?

Reply:
why we maintain two exporter if we can just use grpc unless there is some known limitation that tcp fix but if we have no answer I think we can just use gRPC

Reply:
@jotak @shach33 thoughts ?

Reply:
I feel as @jpinsonneau  mentioned, initially with PCA the support was intended only for tcp. Merging this would make using the tool more structured.  
@msherif1234 The reason to have a it separate from grpc was to export in pcap.

Reply:
with grpc we are still producing pcaps from the CLI repo too right ? my worry is we have a mode that is not used but I am not strongly against

Reply:
If we still continue to export packets as pcap via grpc, i feel we should move forward with the change.

Reply:
@jpinsonneau pls confirm ?

Reply:
Indeed the same bytes are sent either with TCP or gRPC
both produces pcap files in the CLI

Reply:
I removed tcp and updated example
https://github.com/netobserv/netobserv-ebpf-agent/pull/291/commits/6e65e79a0050842f22a82466d71bba97e02550ed